shared/src/simulation/activities/provide-personnel-from-vehicles.ts

```diff
@@ -153,17 +151,18 @@ function personnelInVehicleTemplate(
     templateId: UUID
 ): {
     vehicleType: string | undefined;
-    vehiclePersonnel: ResourceDescription<PersonnelType>;
+    vehiclePersonnel: ResourceDescription;
 } {
     const resource = cloneDeepMutable(
         PersonnelResource.create()
     ).personnelCounts;
```

Comment:
This resource description is not 0-initialized and thus the `++` leads to `NaN` values that break the activity.